mitodl/ocw-studio
# Add referencing_pages to external resources

### What are the relevant tickets?
Part of https://github.com/mitodl/hq/issues/5736. Adding `referencing_pages` for newly-created or newly-linked external resources will be handled in a follow-up PR.

### Description (What does it do?)

This PR adds the `referencing_pages` field to the WebsiteContent Django model, and also updates the parsing rules for legacy links.

### How can this be tested?

Run 

`docker compose exec web ./manage.py markdown_cleanup link_to_external_resource --skip-sync --commit --filter <course ID>`

and 

`docker compose exec web ./manage.py markdown_cleanup nav_item_to_external_resource --skip-sync --commit --filter <course ID>`

and verify that the `referencing_pages` in Django admin looks as expected.
